### DISCUSSION TITLE
谁该拥有开源项目？人还是公司？美国正在加大开源的采用；等开源之道每周评论 2021 08 29 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#56] [Announcements]
# 谁该拥有开源项目？人还是公司？美国正在加大开源的采用；等开源之道每周评论 2021 08 29 - 「开源之道」

https://opensourceway.community/posts/opensource/daily_reading/2021/%E5%BC%80%E6%BA%90%E4%B9%8B%E9%81%93%E6%AF%8F%E5%91%A8%E8%AF%84%E8%AE%BA-2021-08-29/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

单回答标题,

> 开源拥有开源,

这是 GPL 开创的自反命题最有趣以及最有力量的理解.
